packages/connect-common/files/coins.json

```diff
@@ -38,6 +38,7 @@
             "min_address_length": 27,
             "minfee_kb": 1000,
             "name": "Bitcoin",
+            "overwintered": false,
```

Comment:
I have not modified this file manually, only with the script `scripts/update-coins.sh`. I guess those `"overwintered": false` are new thing or the script needs to be fixed? @mroz22

Reply:
this is ok, they probably updated definitions in trezor-firwmare. overwintered applies only for zcash, other coins have it either undefined or set to false.